pandas/tests/copy_view/test_chained_assignment_deprecation.py

```diff
@@ -33,117 +32,11 @@ def test_methods_iloc_warn(using_copy_on_write):
-    df = df_orig.copy()
-    ser = df.copy()["a"]
-    getattr(ser, func)(*args, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    ser = df.iloc[:, 0]  # iloc creates a new object
-    getattr(ser, func)(*args, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    ser = df["a"]
-    getattr(ser, func)(*args, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    # TODO(CoW-warn) because of the usage of *args, this doesn't warn on Py3.11+
-    if using_copy_on_write:
-        with tm.raises_chained_assignment_error(not PY311):
-            getattr(df["a"], func)(*args, inplace=True)
-    else:
-        with tm.assert_cow_warning(not PY311, match="A value"):
-            getattr(df["a"], func)(*args, inplace=True)
-
-    df = df_orig.copy()
-    ser = df["a"]  # populate the item_cache and keep ref
-    if using_copy_on_write:
-        with tm.raises_chained_assignment_error(not PY311):
-            getattr(df["a"], func)(*args, inplace=True)
-    else:
-        # ideally also warns on the default mode, but the ser' _cacher
-        # messes up the refcount + even in warning mode this doesn't trigger
-        # the warning of Py3.1+ (see above)
-        with tm.assert_cow_warning(warn_copy_on_write and not PY311, match="A value"):
-            getattr(df["a"], func)(*args, inplace=True)
-
-
-def test_methods_iloc_getitem_item_cache_fillna(
-    using_copy_on_write, warn_copy_on_write
-):
-    # ensure we don't incorrectly raise chained assignment warning because
-    # of the item cache / iloc not setting the item cache
-    df_orig = DataFrame({"a": [1, 2, 3], "b": 1})
-
-    df = df_orig.copy()
-    ser = df.iloc[:, 0]
-    ser.fillna(1, inplace=True)
-
-    # parent that holds item_cache is dead, so don't increase ref count
-    df = df_orig.copy()
-    ser = df.copy()["a"]
-    ser.fillna(1, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    ser = df.iloc[:, 0]  # iloc creates a new object
-    ser.fillna(1, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    ser = df["a"]
-    ser.fillna(1, inplace=True)
-
-    df = df_orig.copy()
-    df["a"]  # populate the item_cache
-    if using_copy_on_write:
-        with tm.raises_chained_assignment_error():
-            df["a"].fillna(1, inplace=True)
-    else:
-        with tm.assert_cow_warning(match="A value"):
-            df["a"].fillna(1, inplace=True)
-
-    df = df_orig.copy()
-    ser = df["a"]  # populate the item_cache and keep ref
-    if using_copy_on_write:
-        with tm.raises_chained_assignment_error():
-            df["a"].fillna(1, inplace=True)
-    else:
-        # TODO(CoW-warn) ideally also warns on the default mode, but the ser' _cacher
-        # messes up the refcount
-        with tm.assert_cow_warning(warn_copy_on_write, match="A value"):
-            df["a"].fillna(1, inplace=True)
```

Comment:
We might want to keep this part, as there is some behaviour that is relevant for the warning in true CoW mode as well (not only for the warning mode), xref https://github.com/pandas-dev/pandas/issues/56456